src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_pending_resumption_handler.cc

```diff
@@ -0,0 +1,292 @@
+#include "rc_rpc_plugin/rc_pending_resumption_handler.h"
+#include "rc_rpc_plugin/rc_helpers.h"
+#include "rc_rpc_plugin/rc_module_constants.h"
+
+namespace rc_rpc_plugin {
+
+CREATE_LOGGERPTR_GLOBAL(logger_, "RCPendingResumptionHandler")
+
+RCPendingResumptionHandler::RCPendingResumptionHandler(
+    application_manager::ApplicationManager& application_manager)
+    : ExtensionPendingResumptionHandler(application_manager)
+    , rpc_service_(application_manager.GetRPCService()) {}
+
+void RCPendingResumptionHandler::on_event(
+    const application_manager::event_engine::Event& event) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  sync_primitives::AutoLock lock(pending_resumption_lock_);
+  // remove saved resumption request
+  // if event success
+  //    find freazed resumptions that covered by event
+  //    raize events for this freezed resumptions
+  // freezed_subscription = freezed.pop
+  // request_ref = CreateResumptionRequest(freezed_subscription)
+  //      request_ref[application_manager::strings::params]
+  //                 [application_manager::strings::correlation_id]
+  //                    .asUInt();
+  // auto resumption_request = MakeResumptionRequest(corr_id, function_id,
+  // *request);
+  // save (resumption_request)
+  // subscribe_on_event(function_id, corr_id);
+  auto cid = event.smart_object_correlation_id();
+  if (pending_requests_.find(cid) == pending_requests_.end()) {
+    LOG4CXX_DEBUG(logger_, "correlation id: " << cid << " NOT found");
```

Comment:
This is a warning or an error. It means that we subscribed to the event of the message that was not sent. 

ANd make more clear message like : 

'Not waiting for message with correlation id : '